src/Repositories/SnapshotRepository.php

```diff
@@ -25,11 +24,9 @@ public function __construct(
     /**
      * Checks if the snapshot exists.
      */
-    public function has(TestCase $testCase, string $description): bool
+    public function has(): bool
```

Comment:
@nunomaduro maybe we shall rename this one to `exists`?

Reply:
@fabio-ivona Yes.